### ISSUE TITLE
Running a rule against all issues every day

### ISSUE THREAD

**Issue by SD10** [danger/peril#227]
I'd like to create a rule that closes stale issues like [stalebot](https://github.com/probot/stale). 

Is peril capable of repeating a task on all issues, say, every 24 hours?

**Comment by orta** [comment]
Yep, [the scheduler can do that](https://github.com/danger/peril/blob/78312185e13b0d7ece81752fb4239ad8dff3f6b7/docs/settings_repo_info.md#scheduler)
